src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -65,6 +65,9 @@ def detect_sdk_crash(
                 },
             )
 
+            # So Sentry can tell how many projects are impacted by this SDK crash
+            set_path(sdk_crash_event_data, "user", "id", value=event.project.id)
```

Comment:
If we're only interested in the _number_ of users (not tracking those users down), should we hash the project id here, to be on the safe side of any privacy concerns?

Reply:
We already store the project id here https://github.com/getsentry/sentry/blob/a54d21244aae759c131638470242b676dcece849/src/sentry/utils/sdk_crashes/sdk_crash_detection.py#L62-L65

There is nothing you can do with the project id. Only with admin rights in Sentry you can find and access the project. Why do you think there are any privacy concerns?

Reply:
OK, missed that. I may have been thinking too far ahead -- if this ever becomes a more general "sentry for library developers" feature and we make these errors public, we have to make sure that this user ID cannot be traced back to the original organization. As long as this is our own private feature, this should be fine though.

Reply:
Excellent that you think ahead, but this is not a problem right now.